templates/telco/index.html

```diff
@@ -43,836 +38,545 @@ <h1>
-      <p>
-        Run and remotely manage physical, virtualised and containerised workloads in a fully automated way with a minimal footprint.
-      </p>
-      <p>
-        <a class="p-link--external" href="https://maas.io">MAAS</a>, <a class="p-link--external" href="https://linuxcontainers.org">LXD</a>, <a class="p-link" href="/ceph">Ceph</a> and <a class="p-link--external" href="https://microk8s.io">MicroK8s</a>. Proper tools for serious workloads.
-      </p>
-      <p>
-        <a></a>
+        <a href="/cnf">Read more&nbsp;&rsaquo;</a>
       </p>
     </div>
-    <div class="col-6 u-align--center u-vertically-center u-hide--small">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/76174475-pasted+image+0.png",
+    <div class="col-6 u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/f145e15a-K8s_Multi-cloud_solid.svg",
         alt="",
-        height="280",
-        width="460",
+        width="252",
+        height="150",
         hi_def=True,
-        loading="auto"
+        loading="lazy"
         ) | safe
       }}
     </div>
+    <hr class="p-separator" />
   </div>
-</section>
-
-<section class="p-strip--light">
   <div class="row">
-    <div class="col-5 u-align--center u-vertically-center u-hide--small">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/d40fc8b9-Internet+of+Things_digital-gateways.svg",
+    <div class="col-6 u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/669390e0-openstack-cloud.svg",
         alt="",
-        height="280",
-        width="320",
+        width="263",
+        height="150",
         hi_def=True,
-        loading="auto"
+        loading="lazy"
         ) | safe
       }}
     </div>
-    <div class="col-7">
-      <h2 class="u-sv3">Far edge: gateways, networking, digital signage and more</h2>
-      <p>
-        Avoid device lock and benefit from fully automated updates and operations by using an embedded version of the Ubuntu operating system - <a href="/core">Ubuntu Core</a>.
-      </p>
-      <p>
-        Gateways, network devices and digital signage. Designed for devices with limited hardware resources, Ubuntu Core is both lightweight and robust.
-      </p>
-      <p class="u-sv2">
-        Fully hardened OS. Automated remote updates. Both virtualised and containerised workloads supported.
-      </p>
+    <div class="col-6">
+      <h2>Infrastructure for NFV</h2>
+      <h4>OpenStack engineered for best price–performance</h4>
+      <p>Designed to be economical in every way.</p>
+      <p>Smart operations, optimal architecture, better pricing.</p>
+      <p>All of this to ensure TCO reduction of your cloud, while expanding your budget for innovation.</p>
       <p>
-        <a  href="/engage/ubuntu-core-security-audit">Read the whitepaper: "Ubuntu Core: a cybersecurity analysis"&nbsp;&rsaquo;</a>
+        <a href="/vnf">Read more&nbsp;&rsaquo;</a>
```

Comment:
Shouldn't this link be "/telco/openstack" instead?

Reply:
The vnf page is talking about openstack, or did you mean to change the URL?

Reply:
Yeah in the navigation you defined VNF like so 
```yaml
    - title: VNF
      path: /telco/openstack
```
so I just wondered if `/vnf` shouldn't be replaced with `/telco/openstack` here